src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -0,0 +1,104 @@
+import React, { useContext, useState } from "react";
+import { ConfigContext } from "Context";
+import { useAsync } from "react-use";
+import Loader from "components/Loader";
+import { FormattedMessage, useIntl } from "react-intl";
+import { v4 as uuidv4 } from "uuid";
+import {
+  ButtonGroup,
+  Heading,
+  LinkButton,
+} from "@utrecht/component-library-react";
+import { get, put } from "../../api";
+import PropTypes from "prop-types";
+
+const propTypes = PropTypes.shape({
+  heading: PropTypes.string,
```

Comment:
```suggestion
  heading: PropTypes.node,
```
React elements aren't strings, but they can be rendered, so this is a more appropriate prop type (see the default prop)